core/templates/dev/head/pages/privacy-page/privacy-page.mainpage.html

```diff
@@ -1,436 +1,435 @@
-        information removed.
-      </p>
-
-      <a class="oppia-privacy-terms-anchor" name="jurisdictional-issues"></a>
-      <h2>Jurisdictional issues</h2>
-      <p>
-        The Site is controlled and operated by us from the United States, and
-        is not intended to subject us to the laws or jurisdiction of any state,
-        country or territory other than that of the United States. Your
-        Personal Information may be stored and processed in any country where
-        we or our service providers have facilities, and by using the Site you
-        consent to the transfer of information to countries outside of your
-        country of residence, including the United States, which may have
-        different data protection rules than those of your country. In certain
-        circumstances, courts, law enforcement agencies, regulatory agencies or
-        security authorities in those other countries may be entitled to access
-        your Personal Information.
-      </p>
-
-      <a class="oppia-privacy-terms-anchor" name="links-to-third-party-sites"></a>
-      <h2>Links to third-party sites</h2>
-      <p>
-        Our Site and content may feature links to third party websites that
-        offer goods, services or information. When you click on one of these
-        links, you will be leaving our Site and will no longer be subject to
-        this Privacy Policy. We are not responsible for the information or other
-        practices of the other sites that you visit and urge you to review
-        their privacy policies before you provide them with any personally
-        identifiable information. Third party sites or services may collect and
-        use information about you in a way that is different from this Privacy
-        Policy.
-      </p>
-
-      <a class="oppia-privacy-terms-anchor" name="changes-and-updates-to-this-privacy-policy"></a>
-      <h2>Changes and updates to this Privacy Policy</h2>
-      <p>
-        Oppia will continue to evolve and we may update our Privacy Policy from
-        time to time to reflect our practices. The "Last updated" legend at the
-        bottom of this Privacy Policy indicates when this Privacy Policy was
-        last revised, and any changes will become effective immediately upon
-        posting of the Privacy Policy on the Site. Your continued use of the
-        Site following these changes constitutes your agreement to the revised
-        Privacy Policy.
-      </p>
-
-      <a class="oppia-privacy-terms-anchor" name="contacting-us"></a>
-      <h2>Contacting Us</h2>
-      <p>
-        If you have questions about this Privacy Policy or would like to
-        provide feedback, please send an e-mail to admin@oppia.org.
-      </p>
-
-      <em>Last updated: 24 May 2018</em>
-    </md-card>
-  </div>
-{% endblock %}
-
-{% block footer %}
-  {% include 'pages/oppia_footer_directive.html' %}
-{% endblock %}
+      </navbar-breadcrumb>
```

Comment:
Move the `<navbar-breadcrumb>` before the `<content>`

Reply:
Done